### ISSUE TITLE
Error: converting tfplan to CAI assets: adding resource changes to converter: adding resource create or update augmenting asset: getting resource ancestry for project prj-c-billing-logs-111c: googleapi: Error 403: The caller does not have permission

### ISSUE THREAD

**Issue by GuruprasadK0105** [terraform-google-modules/terraform-example-foundation#620]
### TL;DR

Hi Team,

I am using cloud build steps to create a GCP test ORG. I am getting below error while completing 1-ORG step.
Could you please help.

ERROR: build step 3 "us-central1-docker.pkg.dev/prj-b-cicd-b9a7/prj-tf-runners/terraform" failed: step exited with non-zero status: 33
ERROR
Finished Step #3 - "tf validate"
Step #3 - "tf validate": 
Step #3 - "tf validate":       --verbose   Log output to stderr
Step #3 - "tf validate": Global Flags:
Step #3 - "tf validate": 
Step #3 - "tf validate":       --project string       Provider project override (override the default project configuration assigned to the google terraform provider when validating resources)
Step #3 - "tf validate":       --policy-path string   Path to directory containing validation policies
Step #3 - "tf validate":       --output-json          Print violations as JSON
Step #3 - "tf validate":       --offline              Do not make network requests
Step #3 - "tf validate":   -h, --help                 help for validate
Step #3 - "tf validate":       --ancestry string      Override the ancestry location of the project when validating resources
Step #3 - "tf validate": Flags:
Step #3 - "tf validate": 
Step #3 - "tf validate":   terraform-validator validate <tfplan> [flags]
Step #3 - "tf validate": Usage:
Step #3 - "tf validate": Error: converting tfplan to CAI assets: adding resource changes to converter: adding resource create or update augmenting asset: getting resource ancestry for project prj-c-billing-logs-111c: googleapi: Error 403: The caller does not have permission
Step #3 - "tf validate": ERROR: logging before flag.Parse: I0107 10:10:12.292235     120 convert.go:207] unknown resource: random_string
Step #3 - "tf validate": ERROR: logging before flag.Parse: I0107 10:10:12.292225     120 convert.go:207] unknown resource: random_id
Step #3 - "tf validate": ERROR: logging before flag.Parse: I0107 10:10:12.292210     120 convert.go:213] unsupported resource: google_service_account
Step #3 - "tf validate": ERROR: logging before flag.Parse: I0107 10:10:12.292203     120 convert.go:213] unsupported resource: google_project_default_service_accounts
Step #3 - "tf validate": ERROR: logging before flag.Parse: I0107 10:10:12.292195     120 convert.go:213] unsupported resource: google_billing_budget
Step #3 - "tf validate": ERROR: logging before flag.Parse: I0107 10:10:12.292187     120 convert.go:207] unknown resource: random_id

### Expected behavior

Successfully complete 1-ORG step.

### Observed behavior

Step #3 - "tf validate": Error: converting tfplan to CAI assets: adding resource changes to converter: adding resource create or update augmenting asset: getting resource ancestry for project prj-c-billing-logs-111c: googleapi: Error 403: The caller does not have permission

### Terraform Configuration

```hcl
Step #3 - "tf validate": Error: converting tfplan to CAI assets: adding resource changes to converter: adding resource create or update augmenting asset: getting resource ancestry for project prj-c-billing-logs-111c: googleapi: Error 403: The caller does not have permission
Step #3 - "tf validate": ERROR: logging before flag.Parse: I0107 10:10:12.292235     120 convert.go:207] unknown resource: random_string
```

### Terraform Version

```sh
guru@cloudshell:~ (prj-b-cicd-b9a7)$ terraform version
Terraform v1.0.11
on linux_amd64

Your version of Terraform is out of date! The latest version
is 1.1.2. You can update by downloading from https://www.terraform.io/downloads.html

guru@cloudshell:~ (prj-b-cicd-b9a7)$ uname -a
Linux cs-557244416187-default-boost-47rzb 5.10.68+ #1 SMP Wed Dec 1 10:07:21 UTC 2021 x86_64 GNU/Linux
```

### Additional information

No

**Comment by amendezglobant** [comment]
Hello I have the same problem
Error: converting tfplan to CAI assets: adding resource changes to converter: adding resource create or update augmenting asset: getting resource ancestry for project prj-c-billing-logs-0843: googleapi: Error 403: The caller does not have permission

Could you fix this problem?

**Comment by bharathkkb** [comment]
fixed by https://github.com/terraform-google-modules/terraform-example-foundation/pull/629 and long term fix tracked in https://github.com/terraform-google-modules/terraform-example-foundation/issues/546
